commands/doctor.go

```diff
@@ -137,7 +140,7 @@ func (cmd *Doctor) Run(c *cli.Context) error {
 		output, err := exec.Command("docker-machine", "ssh", cmd.machine.Name, "df -h 2> /dev/null | grep /dev/sda1 | head -1 | awk '{print $5}' | sed 's/%//'").Output()
 		if err == nil {
 			dataUsage := strings.TrimSpace(string(output))
-			if i, err := strconv.Atoi(dataUsage); err == nil {
+			if i, e := strconv.Atoi(dataUsage); e == nil {
```

Comment:
Why e?

Reply:
The linter was giving warnings about the second `err` shadowing the one from it's parent scope.